internal/syncronizedmap/syncronizedmap.go

```diff
@@ -0,0 +1,54 @@
+package syncronizedmap
+
+import "sync"
+
+//FIXME: add UT
+type SyncronizedKernelOsDtkMapping interface {
+	NewNodeInfo(kernelVersion, osImage string)
+	NewImageStreamInfo(osImage, dtkImage string)
+	GetImage(kernelVersion string) (string, bool)
+}
+
+type syncronizedKernelOsDtkMapping struct {
+	m          *sync.RWMutex
```

Comment:
IMHO, no need to block writes into `osToDtk` when we are writing to `kernelToOs`. I would have two mutexes here.
Or maybe only one map? WDYT?

Reply:
https://github.com/rh-ecosystem-edge/kernel-module-management/pull/96#discussion_r1020243279
This is what I think :)